fireflight/src/index.js

```diff
@@ -1,20 +1,22 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App";
+import * as serviceWorker from "./serviceWorker";
```

Comment:
Is there a reason you change all these ' to "?

Reply:
No, my VS code auto formats to double quotes.